MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,12 +20,16 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorState: Observer<String?>?
```

Comment:
`onErrorState` could be `onErrorStateChange` to match the `onLoadingStateChange` property.

```suggestion
	var onErrorStateChange: Observer<String?>?
```